school_fees/report/report_student_fees_register.py

```diff
@@ -0,0 +1,30 @@
+# See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+
+class ReportStudentFeesRegister(models.AbstractModel):
+    _name = "report.school_fees.student_fees_register"
+    _description = "School Fees Register Report"
+
+    def get_month(self, indate):
+        """Method to get month"""
+        out_date = indate.strftime("%B") + "-" + indate.strftime("%Y")
+        return out_date
```

Comment:
No need of out_date variable, direct use 
return indate.strftime("%B") + "-" + indate.strftime("%Y")